src/features/speciesfeature.jl

```diff
@@ -48,6 +48,7 @@ function SpeciesFeatureDescriptor(name::String)
     else
         # TODO: figure out default binning situation for continuous-valued SFD's
         #codec = OneHotOneCold(false, )
+        codec = DirectCodec(1)
```

Comment:
does this actually make sense as a default?

Reply:
I doubt it, but I needed something there for the time being. Ideally we would have have to get a number of bins from the user or guess. I don't think OneHotOneCold would work since that implies categorical variable, which continuous variables are not.

Reply:
`OneHotOneCold` works for categorical variables, you just have to specify bins. That's what we've been doing all along since the `DirectCodec` was just added...

Reply:
Right. DataFrames does some kind of trickery to inspect whether a column is categorical or notz but I don't think we need to be doing anything too sophisticated at this stage. I was thinking assuming something is a continuous variable as the default would be reasonable and if a feature is categorical that would need to be described explicitly.